### PR TITLE
Add some profiler sections

### PR DESCRIPTION
**Pull request by cconard96**
Add GLPI profiler sections for some Symfony config providers and custom asset capacity bootstrapping.

### CODE REVIEW

**Comment by trasher** [comment]
e2e tests are failing, no idea if it's related

**Review by orthagh — APPROVED** [review]
It works great !

File: src/Glpi/Debug/Profiler.php [hunk 1]
```diff
@@ -51,6 +51,8 @@ final class Profiler
     public const CATEGORY_DB = 'db';
     public const CATEGORY_TWIG = 'twig';
     public const CATEGORY_SEARCH = 'search';
+    public const CATEGORY_CUSTOMOBJECTS = 'customobjects';
+    public const CATEGORY_SYMFONY = 'symfony';
```
Comment (cedric-anne) [inline src/Glpi/Debug/Profiler.php:55]: This is not really related to Symfony itself. Indeed, everything in this section is actually related to the GLPI framework initialization / bootstrapping.

```suggestion
    public const CATEGORY_GLPI_BOOT = 'glpi-boot';
```

**Review by cedric-anne — APPROVED** [review]
I renamed the category from `symfony` to `boot`.
